tslint.json

```diff
@@ -3,6 +3,12 @@
     "tslint-microsoft-contrib"
   ],
   "rules": {
+    "newline-per-chained-call": false,
+    "triple-equals": false,
```

Comment:
Is disabling these rules necessary? Would it be better to fix our code to make ts-lint happy?
triple-equals in particular is a useful one, e.g. I saw a potential bug around "/mutate" comparison that would've been caught by it.

Reply:
Okay, I can do it. I was not sure if I would obey the ts-lint or follow the denial pattern of who created the lib :P
So I decided to add the rules to lower the alerts.

But now I'm going to work on it.

Reply:
done